app/src/main/java/com/google/droidjump/Droid.java

```diff
@@ -20,34 +20,40 @@
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 
+/**
+ * Represents Droid character.
+ */
 public class Droid extends DrawableElement {
 
     private boolean isJumping;
     private boolean isCrouching;
     private Bitmap[] droidTypes;
     private int initialY;
     private int jumpHeight;
-    // so that droid can easily jump over all obstacles we need to add two measures: the highest obstacle + additional height
-    private final static int additionalHeight = 50;
     // the variable is responsible for how much time ticks full animation of droid goes
     public final static int fullAnimationTicks = 4;
     // the variable is responsible for how much time ticks must pass to change a bitmap so that animate droid smoothly
     public final static int animationStepTicks = 2;
-
+    // so that droid can easily jump over all obstacles we need to add two measures: the highest obstacle + additional height
+    private final static int additionalHeight = 50;
 
     public Droid(int x, int y, Resources resources) {
```

Comment:
I am confused, why don't we use this why for this.y; where is the assignment?

Reply:
synced offline and agreed to resolve comment